sky/clouds/service_catalog/constants.py

```diff
@@ -1,5 +1,5 @@
 """Constants used for service catalog."""
-HOSTED_CATALOG_DIR_URL = 'https://raw.githubusercontent.com/skypilot-org/skypilot-catalog/master/catalogs'  # pylint: disable=line-too-long
+HOSTED_CATALOG_DIR_URL = 'https://skypilot-catalog.s3.us-east-1.amazonaws.com/catalogs'  # pylint: disable=line-too-long
```

Comment:
minor: could we have a fallback list, e.g. use the github link first and fallback to this mirror?